invenio_jobs/logging/celery_signals.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2025 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Celery signals."""
+
+from celery import signals
+
+from .jobs import job_context
+
+
+# Capture context when a task is sent
+@signals.before_task_publish.connect
```

Comment:
add comment that this is only executed by async tasks while the below are executed by all taks